derp/derp_server.go

```diff
@@ -878,6 +880,9 @@ func (c *sclient) handleFrameForwardPacket(ft frameType, fl uint32) error {
 		reason := dropReasonUnknownDestOnFwd
 		if dstLen > 1 {
 			reason = dropReasonDupClient
+		} else {
+			// TODO: rate-limit
+			go c.requestPeerGoneWrite(dstKey, PeerGoneReasonNotHere)
```

Comment:
I think we should probably only send this type of DERP frame on disco packets (CallMeMaybe messages), not on WireGuard packets.

Reply:
So always test the packet to see if it is CallMeMaybe and only send if it is? Any thoughts on rate-limiting, should that be per server, per client, per false destination, something else?

Reply:
It now checks for CallMeMaybe with LooksLikeDiscoWrapper() and sends only 3 PeerGone packets per second.